app/(gcforms)/[locale]/(form filler)/id/[...props]/actions.ts

```diff
@@ -13,6 +13,7 @@ import { verifyHCaptchaToken } from "@clientComponents/globals/FormCaptcha/actio
 import { checkOne } from "@lib/cache/flags";
 import { FeatureFlags } from "@lib/cache/types";
 import { validateResponses } from "@lib/validation/validation";
+import { sendNotification } from "./lib/notifications";
```

Comment:
if the notifications.ts file is moved to the `/lib` folder then this just needs to be updated to `@lib/notifications`